src/DotNetWorker.Core/DotNetWorker.Core.csproj

```diff
@@ -26,6 +26,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="System.Collections.Immutable" Version="8.0.0" />
+    <PackageReference Include="Microsoft.Bcl.Memory" Version="9.0.0" />
```

Comment:
We may need to discuss. Bringing this in so we have the ability to use range patterns in our .NET standard targets. This package will provide implementations of `Index` and `Range`, required for this to work. 9.0.0 is the first version of this package, but it does introduce dependencies.

Reply:
Are `Range` and `Index` in our app merely syntax sugar, or do they offer any performance benefits? If they are just syntax sugar, should we consider avoiding their use to prevent adding an extra assembly? Reducing this dependency could also help minimize the publish payload for the function app.

Reply:
>  Reducing this dependency could also help minimize the publish payload for the function app.

There is no impact to the payload size for TFMs that do not have a dependency on this (in this case, this dependency would only come in for .NET standard/NETFX scenarios).

This impacts anything that relies on the functionality those types provide, and some of the language syntax does.

Reply:
> Reducing this dependency could also help minimize the publish payload for the function app.

There will be no payload size impact on the TFMs that have no dependency on this (this reference is conditional, for .NET Standard only). 

Those types bring functionality on their own, but some language syntax also has a dependency on them.